docs/User Interfaces.md

```diff
@@ -16,89 +49,109 @@ Next, create your sub-applications for Nerves and for Phoenix:
 ```bash
 $ cd nervy/apps
 $ mix nerves.new fw
-$ mix phoenix.new ui --no-ecto --no-brunch
+$ mix phx.new ui --no-ecto --no-brunch
 ```
 
-Now, add the Phoenix `ui` app and the `nerves_networking` library to the `fw` app as dependencies:
+Now, add the Phoenix `ui` app and the `nerves_network` library to the `fw` app as dependencies:
 
 ```elixir
-# nervy/apps/fw/mix.exs
+# apps/fw/mix.exs
 
-...
-defp deps do
+# ...
+def deps do
   [{:ui, in_umbrella: true},
-   {:nerves_networking, "~> 0.6.0"}]
+   {:nerves_network, "~> 0.3"}]
 end
-...
+# ...
+```
+
+##### Specifying configuration order
+
+By default, the main config loads the application configs in an unordered way: `import_config "../apps/*/config/config.exs`. This can cause problems if the  `ui` config is applied last: we may lose overrides applied in the `fw` config. You need to force the order in which the config files get imported:
+
+```elixir
+# nervy/config/config.exs
```

Comment:
It seems like you prefer to not mention `nervy` in the file path (which is fine with me). Let's be consistent and not include it here either.

Reply:
Sure, I initially included it here because it could otherwise be confusing which `config` folder was being referenced within an umbrella project layout. But I agree it's not as consistent.